File: baekjoon/05_Array/01_10818/10818.py
import sys

n = int(sys.stdin.readline())
data = list(map(int, sys.stdin.readline().split())) # list

# int
max = data[0]
min = data[0]

for d in data:
    if max < d:
        max = d
    elif d < min:
        min = d
    else:
        pass

print(min, max)

File: C/2022-12-19/10818.c
#include <stdio.h>

int n;

int main() {
    scanf("%d", &n);
    int arr[n];

    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int min = arr[0];
    int max = arr[0];
    for (int i = 0; i < n-1; i++) {
        min = min < arr[i+1] ? min : arr[i+1];
        max = arr[i+1] < max ? max : arr[i+1];
    }

    printf("%d %d", min, max);
}
